x-pack/plugins/ml/public/application/trained_models/models_management/test_models/test_flyout.tsx

```diff
@@ -6,18 +6,47 @@
  */
 
 import * as estypes from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
-import React, { FC } from 'react';
+import React, { FC, useState, useEffect } from 'react';
 
 import { FormattedMessage } from '@kbn/i18n-react';
-import { EuiFlyout, EuiFlyoutHeader, EuiTitle, EuiFlyoutBody, EuiSpacer } from '@elastic/eui';
+import {
+  EuiFlyout,
+  EuiFlyoutHeader,
+  EuiTitle,
+  EuiFlyoutBody,
+  EuiSpacer,
+  EuiTab,
+  EuiTabs,
+  useEuiPaddingSize,
+} from '@elastic/eui';
 
 import { SelectedModel } from './selected_model';
+import { INPUT_TYPE } from './models/inference_base';
+import { useTrainedModelsApiService } from '../../../services/ml_api_service/trained_models';
 
 interface Props {
-  model: estypes.MlTrainedModelConfig;
+  modelId: string;
   onClose: () => void;
 }
-export const TestTrainedModelFlyout: FC<Props> = ({ model, onClose }) => {
+export const TestTrainedModelFlyout: FC<Props> = ({ modelId, onClose }) => {
+  const mediumPadding = useEuiPaddingSize('m');
+
+  const trainedModelsApiService = useTrainedModelsApiService();
+  const [inputType, setInputType] = useState<INPUT_TYPE>(INPUT_TYPE.TEXT);
+  const [model, setModel] = useState<estypes.MlTrainedModelConfig | null>(null);
+
+  useEffect(() => {
```

Comment:
```suggestion
  useEffect(funtion fetchModel() {
```

Reply:
Updated in https://github.com/elastic/kibana/pull/144629/commits/4d468e3fd2287ed24cb78ab13320a3a3c38b828f